auth/auth.go

```diff
@@ -137,6 +138,7 @@ func NewClient(ctx context.Context, conf *internal.AuthConfig) (*Client, error)
 	hc.CreateErrFn = handleHTTPError
 	hc.Opts = []internal.HTTPOption{
 		internal.WithHeader("X-Client-Version", fmt.Sprintf("Go/Admin/%s", conf.Version)),
+		internal.WithHeader("x-goog-api-client", fmt.Sprintf("gl-go/%s fire-admin/%s", runtime.Version(), conf.Version)),
```

Comment:
The output of `runtime.Version()` has a prefix. Example: `go1.21` https://pkg.go.dev/runtime#Version

Would this affect parsing in our regex?

Reply:
Good catch! I think that should be okay as it will be consistent everywhere. Let's confirm this offline with Vinay. Thanks!

Reply:
Removed the prefix. Thanks!